src/main/java/com/denys/toukdemo/entity/MovieEntity.java

```diff
@@ -0,0 +1,102 @@
+package com.denys.toukdemo.entity;
+
+import javax.persistence.*;
+import java.sql.Date;
+import java.util.Collection;
+
+@Entity
+@Table(name = "movie", schema = "public", catalog = "postgres")
+public class MovieEntity {
+    private Integer id;
+    private String title;
+    private String description;
+    private Date premierStart;
+    private Date premierEnd;
+    private Collection<SeansEntity> seansById;
+
+    @Id
+    @GeneratedValue
+    @Column(name = "id", nullable = false)
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    @Basic
+    @Column(name = "title", nullable = false, length = 100)
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    @Basic
```

Comment:
po co ta adnotacja?

Reply:
> po co ta adnotacja?

skorzystałem z pluginu w Intellij do generowania Entity w pierwszej iteracji, 
Dokumentacji mówi że ta adnotacja wskazuje najprostszy typ mappingu i jest raczej  tyłko dodatkowym wskazaniem na to że to pole ma być jakimś prostym typem